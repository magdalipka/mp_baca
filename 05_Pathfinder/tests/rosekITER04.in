5 5
1 1 1 0 0
0 0 0 0 1
1 1 0 1 1
0 1 0 0 1
0 0 0 1 0
1
i 3 1 4 4

