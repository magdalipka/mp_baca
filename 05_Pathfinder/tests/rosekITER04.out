i W N N E N E
